core/qthelper.cpp

```diff
@@ -1143,6 +1143,11 @@ QString localFilePath(const QString &originalFilename)
 	return localFilenameOf.value(originalFilename, originalFilename);
 }
 
+// the water types need to match the watertypes enum
+const QStringList waterTypes = {
+	gettextFromC::tr("Fresh"), gettextFromC::tr("Salty"), gettextFromC::tr("EN13319"), gettextFromC::tr("Salt"), gettextFromC::tr("Use DC")
```

Comment:
i guess the original `use dc` was consistent with lines like:
https://github.com/subsurface/subsurface/blob/649b2f2a9ea581e4fbb97a6fd8cc519af880aaf2/desktop-widgets/tab-widgets/TabDiveInformation.cpp#L26

so we should either change those too or keep them all `use dc`.

Reply:
I think keeping them all as ``use dc`` for now keeps the change targetted. But I could see another change to change them all.